### PR TITLE
Yash_Assignment_19_110 mins_Done

### PR DESCRIPTION
**Pull request by YashCJun20**
Outputs ---->
Program 1 
image

Program 2
image

Program 3
image

Program 4
image

Program 5
image

### CODE REVIEW

File: src/yashCJun20/Assignment19/FindIndex.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package yashCJun20.Assignment19;
+
+public class FindIndex {
+	
+	void toFindDifferIndex (int [] arr1,int [] arr2) {
+		
+		for (int i=0;i<arr1.length;i++) {
+		
+        	if (arr1[i]!=arr2[i])
```
Comment (KrishnaKTechnocredits) [inline src/yashCJun20/Assignment19/FindIndex.java]: What is the difference between DifferIndex class and FindIndex class.

Comment (KrishnaKTechnocredits) [reply src/yashCJun20/Assignment19/FindIndex.java]: Give Proper method name.

File: src/yashCJun20/Assignment19/StringArray.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package yashCJun20.Assignment19;
+
+public class StringArray {
+	
+	void toFindAllDifferIndex (String [] arr1,String [] arr2){
+		
+		for (int i=0;i<arr1.length;i++) {
+			
+        	if (arr1[i]!=arr2[i]) {
```
Comment (KrishnaKTechnocredits) [inline src/yashCJun20/Assignment19/StringArray.java]: You should use .equals() method for String comparison.

File: src/yashCJun20/Assignment19/FindIndex.java (hunk 1 above)
Comment (YashCJun20) [reply src/yashCJun20/Assignment19/FindIndex.java]: As advised,changes made to the code.
Please review.
I will take care of this for the upcoming assignments

File: src/yashCJun20/Assignment19/StringArray.java (hunk 2 above)
Comment (YashCJun20) [reply src/yashCJun20/Assignment19/StringArray.java]: As advised,changes made to the code.
Please review.
I will take care of this for the upcoming assignments

**Pull request merged**